### ISSUE TITLE
Accessibility - Visible label and accessible name do not match

### ISSUE THREAD

**Issue by id2win** [GSA/idmanagement.gov#514]
Issue Level A

Why?
Using two different names for a single page element can create a confusing user experience for assistive technology users.

Ask - Please make appropriate adjustments in text to provide better context for assistive tech users. Current text and recommended text is provided on this link for the 6 occurrences. Recommended text is where we need to be, already confirmed with section 508 team. 

Findings - https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?PageId=38246420896&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r14/failed/U1AwOEVxMmc=;ODRjOWY3MTA=
